Update viralsweep.md (#6935)
braze-inc/braze-docs: _docs/_partners/message_orchestration/channel_extensions/loyalty/viralsweep.md

- Tracking campaign name

### Does ViralSweep update subscribers?

> Yes, if you run a promotion and we pass someone to Braze, then you run another promotion in the future and the same person enters, we will automatically update their information in Braze (if any new information is provided). Mainly, the referral URL will be updated with the newest URL for each promotion they enter, and the promotion ID field will contain the ID of all the promotions they have entered.

Reviewer: ```suggestion
Yes. If you run a promotion and ViralSweep passes someone to Braze, then you run another promotion in the future and the same person enters, that person's information will automatically be updated in Braze (if any new information is provided). Mainly, the referral URL will be updated with the newest URL for each promotion they enter, and the promotion ID field will contain the ID of all the promotions they have entered.
```
